web_accounts_list.json

```diff
@@ -4263,6 +4263,62 @@
       "known_accounts" : ["summit1g","cohhcarnage"],
       "category" : "gaming",
       "valid" : false
-      }
+      },
+      {
+        "name" : "fansly",
+        "check_uri" : "https://apiv2.fansly.com/api/v1/account?usernames={account}",
+        "pretty_uri": "https://fansly.com/{account}/posts",
+        "account_existence_code" : "200",
+        "account_existence_string" : "username",
+        "account_missing_string" : "response: []",
+        "account_missing_code" : "200",
+        "known_accounts" : ["Mikomin","test"],
+        "category" : "XXXPORNXXX",
+        "valid" : true
+        },
+        {
+          "name" : "mym.fans",
+          "check_uri" : "https://mym.fans/{account}",
+          "account_existence_code" : "200",
+          "account_existence_string" : "posts",
+          "account_missing_string" : "Il n’y a rien ici…",
+          "account_missing_code" : "404",
+          "known_accounts" : ["Djelizamay","Andysparkles"],
+          "category" : "social",
+          "valid" : true
+        },
+        {
+          "name" : "SpankPay",
+          "check_uri" : "https://spankpay.me/{account}",
+          "account_existence_code" : "200",
+          "account_existence_string" : "Pay Me",
+          "account_missing_string" : "Error: Bad response status: 404",
+          "account_missing_code" : "200",
+          "known_accounts" : ["Medroxy","Anna"],
+          "category" : "finance",
+          "valid" : true
+        },
+        {
+          "name" : "datezone",
+          "check_uri" : "https://www.datezone.com/users/{account}/",
+          "account_existence_code" : "200",
+          "account_existence_string" : "Profile",
+          "account_missing_string" : "There is an error",
+          "account_missing_code" : "404",
+          "known_accounts" : ["xkarola","zorro3330"],
+          "category" : "dating",

```

Comment:
I know this appears to be a dating site, but looking at some of the blurry profile pictures, those are not safe for work. I can tell what is in those pics. Because of that, please move this to XXXPORNXXX instead of dating.

Reply:
Okay, I wasn't sure how to categorize it.